contrib/IECoreUSD/src/IECoreUSD/AttributeAlgo.cpp

```diff
@@ -151,6 +151,14 @@ IECore::InternedString IECoreUSD::AttributeAlgo::nameFromUSD( IECoreUSD::Attribu
 			{
 				newPrefix = "ai";
 			}
+			else if( prefix == "ai" )
+			{
+				// If the prefix in USD is actually ai:, we can't support it properly - if we pass it through
+				// here, then when we try to look up the value, we would map it back to look for an "arnold:"
+				// prefix, it wouldn't match anything.  This is unlikely to happen - no one should be using
+				// the prefix "ai:", but it could be very confusing.  Give some hint about what has gone wrong.
```

Comment:
This may have already happened - it's exactly what you get if you write Arnold attributes out of Gaffer 0.61. I'm not sure how much we should try to support that or not. If we at least loaded the attributes (see comments in the test), then ShuffleAttributes could be used to shuffle them back to their rightful place in Gaffer.